source/blog/2018-04-22-ember-learning-update.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Community Update from the Learning Team
+author: The Ember Learning Team
+tags: Recent Posts, Learning, Marketing, Roadmap, Community
+responsive: true
+---
+
+Each Ember team meets in person at EmberConf. For many of us, it was our first chance to sit down in person and plan together! This is important for us because it allows us spend an entire day getting to know each other and plan for the upcoming year.
+
+In order to keep the community updated while we move initiatives forward, we'd like to share with you the major outcomes from our face to face meeting. The learning team (and friends!) spent the day reviewing our current workload, extracting all of the ideas from `@locks` brain, and reviewing priorities for the upcoming year.
+
+## Goals
+
+We had a few goals for the day:
+
+- craft a mission statement
+- identify all of the initiatives that were in the scope of the learning team
+- put all of the initiatives into categories
+- prioritize the categories
+
+We were able to identify categories of work that naturally fell in to one of two directions - user support and marketing. Here are some of of those categories, with initiatives for each:
+
+### User Support
+
+- Guides
+- API Docs (version support, modules API, Broccoli)
+- Tutorials (101 Quickstart, 201 Contacts, 301 Super Rentals)
+- Infrastructure (subdomains, Ember-izing the website)
+- Internal Evangelism (office hours, mentoring, simplify contribution process)
+- Tooling and Maintenance (Inspector, Language Servers, Twiddle)
+
+### Marketing
+
+- Website (devsign, upgrades & maintenance)
+- External Evangelism (tech group outreach, podcast & blog collaboration)
+- Traditional Marketing (Blog, Ember Doctrine, Why Ember, Testimonials)
+
+There's more to this list, but we hope this gives everyone a good idea of the direction we are heading. The Ember Learning Team meets every Thursday at 1pm EST. Meeting details are in the topic of the #team-learning channel on Slack- we invite you to join us!
```

Comment:
May be we can link to the meeting notes for those who want to follow along without attending the meetings